### DISCUSSION TITLE
(Feature Request) Improved mobile view

### DISCUSSION THREAD

**Discussion by BeatLink** [TriliumNext/Trilium#8626] [Show and tell]
### Describe feature

I've created some scripts to rework how the mobile view in trillium functions. These scripts allow mobile users to use the full capabilities of the trillium web in desktop mode while still de-cluttering the interface and increasing the size of elements for smaller screens. The scripts can be applied as is to current versions of trillium but perhaps it could be incorporated into the core mobile view of Trilium. 

Give it a try and let me know what you think!

https://github.com/BeatLink/trilium-scripts/tree/main/Mobile%20View

**Comment by meichthys** [comment]
Nice work. I cant wait to try it. What NoteID do we use? Do we use the full path like `#root/kHCVsfPpbpZY/bqHm2hEzJy40/yTPg2v750TiE/BUuf9ATkjoEZ/tNd4ic9aeH5T/angaZoMy9Qc8/5eU9wZ30Cf8K` or just the id listed in the note information pane like: `5eU9wZ30Cf8K` ?

Nevermind, I answered my own question - the shortID works nice 👍

One thing to note: If the side panel is hidden, the `Set Sidebar View` button won't work.

**Comment by BeatLink** [comment]
Yeah, the sidebar view is intended to work with the sidebar kept open

**Comment by LokiLuciferase** [comment]
This is exactly what I needed from the mobile view, thanks!

I think passing the note IDs used in `SetupButton.js` via relations (`api.currentNote.getRelationValue`) is cleaner than putting the actual IDs in the code, but other than that, it worked out of the box like a charm!

**Comment by meichthys** [comment]
Issue migrated to [TriliumNext](https://github.com/TriliumNext/trilium/issues/4962) as future feature requests will be discussed and implemented there.

**Comment by meichthys** [comment]
The mobile editing experience has been dramatically improved in the latest versions of https://github.com/TriliumNext/Notes/
